### ISSUE TITLE
[Request] CVS or formats supported by exel generate rpy and export translations

### ISSUE THREAD

**Issue by Neyunse** [renpy/renpy#3096]
A few days ago, I thought about something that can be complicated to integrate into Ren'Py, and that would be to import Excel generated files to generate rpy files and export/import translations. 

A simple example of my idea would be.
`chapter1.cvs ->  chapter1.rpy`

And the content of the rpy would be generated by obtaining the data and would give this result.
![imagen](https://user-images.githubusercontent.com/36079551/137822773-1a430d4e-b97e-4e59-a3c8-0cd545e7f8c3.png)

`filename -> label <filename>:`

```
label chapter1:
      e "happy time!"
```
If you think it is complicated to include, you can close this request. :D

**Comment by renpytom** [comment]
I don't think a tool like this belongs in Ren'Py, as opposed to being part of a third party tool.

**Comment by Neyunse** [comment]
> I don't think a tool like this belongs in Ren'Py, as opposed to being part of a third party tool.

I guess you're right, internally at renpy I don't think it's a good idea. But outside of it it can be used as a complement. A few months ago I made [this project](https://github.com/KagariSoft/renxel) to "support" this somehow, this tool is just another method to generate a renpy translation. But after seeing the work and thinking about it, I saw that as a third party app this kind of work would be better and not implemented in renpy.
